### ISSUE TITLE
Error in display code while printing validation accuracy and loss

### ISSUE THREAD

**Issue by NiharG15** [tjdevWorks/digit-recogniser-workshop#1]
```

def display_stats(sess, batch_train, batch_train_labels, batch_val, batch_val_labels, cost, accuracy, writer, step):
    """
    Display intermediate results.
    """
    train_data = sess.run([accuracy, summ, cost], feed_dict={
        inputs: batch_train, targets: batch_train_labels, keep_prob: 1.0, learning_rate:0.0001})
    val_data =sess.run([accuracy, cost], feed_dict={
        inputs: batch_val, targets: batch_val_labels, keep_prob: 1.0, learning_rate:0.0001})
    
    writer.add_summary(train_data[1])
    
    print("Step {} Train_Loss : {:>10.4f} Train_Accuracy : {:.6f} Val_Loss : {:>10.4f} Val_Accuracy : {:.6f}"
         .format(step, train_data[2], train_data[0], val_data[0], val_data[1]))
    return
```

In above code, val_data[0] and val_data[1] should be exchanged while printing.
